content/millennium/r4/capability-statement.md

```diff
@@ -0,0 +1,58 @@
+---
+title: CapabilityStatement | R4 API
+---
+
+# CapabilityStatement
+
+* TOC
+{:toc}
+
+## Overview
+
+The CapabilityStatement resource documents the functionality implemented from the HL7<sup>®</sup> FHIR<sup>®</sup> standard. Unlike most resources,
+this resource is found at [`:serviceRootURL/metadata`] instead of by its resource name.
+
+## Metadata
+
+Get the capabilities and configurations of this implementation and deployment of the FHIR standard.
+
+    GET /metadata?:parameters
+    
+_Implementation Notes_
+
+* Authentication is not required to access the Conformance resource
+* No parameters other than the standard `_format` parameter are supported for reading CapabilityStatement
+
+### Authorization Types
+
+Authorization is not required.
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
```

Comment:
Should this be patient: false since patient access isn't currently supported?

Reply:
since auth isn't required, I think this should note that all access types are available. someone can send in a patient token and view capability statement